specs/0111-redesign-profile-page.md

```diff
@@ -0,0 +1,108 @@
+* `Submissions`: translations submitted, including both suggestions and direct translations.
+* `Reviews performed`: reviews performed on pending suggestions.
+* `Reviews received`: reviews received from another translator.
+* `Submissions and reviews`: submissions + reviews performed.
+* `All activities`: submissions + reviews performed + reviews received.
+
+Only one data point is displayed at a time, and it can be selected using the dropdown list above the graph. The default view is `All activities`, since it provides a good picture of the user’s activity in Pontoon.
+
+Each square in the graph represents a day, with color changing depending on the amount of contributions. It’s possible to click on a column to select a specific week, or change the year currently displayed using the links in the rightmost column.
+
+When selecting a specific time range, the data displayed below the graph is updated accordingly. This doesn’t affect the approval and self-approval graphs.
+
+## Other data in the sidebar
+
+The sidebar displays contact information for the contributor:
+* Name.
+* Description.
+* Contact email address.
+* External accounts.
+* Last login.
+* Latest activity.
+* Join date (header `Member since`).
+* Translation rights (header `Translator`), followed by a list of locales for which the user has translator rights. The section is hidden if the user doesn’t have translator rights.
+* Locale manager rights (header `Team  Manager`), followed by a list of locales for which the user has manager rights.
+* A flag indicating if the user is an administrator. The section is hidden if the user doesn’t have manager rights.
+
+## Visibility of data
+
+Most of the data will be visible by default to all users, including non authenticated visitors (i.e. users not logged in to Pontoon).
+
+Visibility for the following fields can be limited by user settings:
+* Email address.
+* External accounts.
+* Self-approval ratio.
+* Approval ratio.
+
+Limited means that non authenticated users, and other contributors, won’t be able to see this information when they visit the profile page. But this information will be always visible to users with translator rights for any locale in Pontoon (translators, locale managers, administrators).
+
+## Dependencies and other changes
+
+### Contact email address
+
+In order to support this new profile, users should be able to provide a contact email address:
+* This is the address that will be displayed in the profile page (pending personal visibility settings), and can be different from the one used to log in to Pontoon.
+* Locale managers and administrators must provide a contact email address, and its visibility can only be public, since contributors need to be able to contact them.
+
+### External accounts
+
+Users should be able to provide a series of external accounts to help locale managers and administrator contacting them:
+* Bugzilla (email address).
+* GitHub (username).
+* Telegram (username).
```

Comment:
The original proposal only had Bugzilla and GitHub.

In the feedback, someone suggested to add Matrix.

Given we often use Telegram, I wonder if it makes sense to add this as well. One concern is about visibility, as we probably want this to be private.

Reply:
Anecdotally speaking optional fields rarely end up being filled out.